new-packaging/README.md

```diff
@@ -0,0 +1,34 @@
+## Pre-Requisites
+You need `xmlstarlet`, `dos2unix`
+
+You also need `cypher-shell` packaged
+
+You can either checkout `cypher-shell` and build it yourself:
+
+`make out/cypher-shell.zip`
+
+Or download the zip file from teamcity
+
+and then,
+
+`cp out/cypher-shell.zip /path/to/neo4j`
+
+## Package `neo4j`
+
+From `/path/to/neo4j/packaging directory` run `mvn install`
```

Comment:
This depends. Do you want people to build neo4j?

If so, it needs to be from `/path/to/neo4j` and not in packaging.

If you don't care if they build neo4j, then the from is fine but you can remove `-DskipCypher` because that only matters when you build neo4j